tools/tool_settings.h

```diff
@@ -927,8 +928,24 @@ static void GetReplayOptions(gfxrecon::decode::ReplayOptions&      options,
         options.override_gpu_index = std::stoi(override_gpu);
     }
 
+    if (arg_parser.IsOptionSet(kSkipFailedAllocationLongOption) ||
+        arg_parser.IsOptionSet(kSkipFailedAllocationShortOption))
+    {
+        options.skip_failed_allocations = true;
+    }
+
+    if (arg_parser.IsOptionSet(kRemoveUnsupportedOption))
+    {
+        options.remove_unsupported_features = true;
+    }
+
     IsForceWindowed(options, arg_parser);
     SetWindowOrigin(options, arg_parser);
+
+    options.screenshot_ranges      = GetScreenshotRanges(arg_parser);
```

Comment:
OXR currently has no screenshot capabilities. (don't know how we'd add without implementing a compositor)

Reply:
Yes, but DX and Vulkan both  shared this code.  But it is now outside of the OpenXR-specific changes.